test/screenshot/text-field/icon/index.tsx

```diff
@@ -15,6 +15,10 @@ const TextFieldIconScreenshotTest = () => {
       <Icon>
         <MaterialIcon tabIndex={0} role='button' icon='favorite' />
       </Icon>
+
```

Comment:
Looks like you added a screenshot test! I'll update once the pr looks good. Thanks!